scripts/retrieve_cutout.py

```diff
@@ -56,20 +54,12 @@
 
     configure_logging(snakemake) # TODO Make logging compatible with progressbar (see PR #102)
```

Comment:
```suggestion
    configure_logging(snakemake)
```

TODO probably obsolete, PR #102 has already been merged for a long time.

Reply:
Scratch that, PR #102 reference is about how `configure_logging` and progressbar don't like each other. Presumably still relevant.